apps/src/blockly/components/ModalFunctionEditor.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+  MODAL_EDITOR_ID,
+  MODAL_EDITOR_NAME_INPUT_ID,
+  MODAL_EDITOR_DELETE_ID,
+  MODAL_EDITOR_CLOSE_ID,
+} from '@cdo/apps/blockly/addons/functionEditorConstants';
+import moduleStyles from './modal-function-editor.module.scss';
+
+export default function ModalFunctionEditor() {
+  return (
+    <div id={MODAL_EDITOR_ID} className={moduleStyles.modalEditor}>
+      <div
+        className="toolbar"
+        style={{
+          zIndex: 1,
+          position: 'absolute',
+          left: '142px',
+          top: '10px',
+          width: '800px',
+        }}
+      >
+        <div style={{float: 'right'}}>
+          <button type="button" id={MODAL_EDITOR_DELETE_ID}>
+            delete
+          </button>
+          <button type="button" id={MODAL_EDITOR_CLOSE_ID}>
+            close
+          </button>
+        </div>
+        <div
+          id={`${MODAL_EDITOR_NAME_INPUT_ID}Container`}
+          style={{padding: '10px', width: '100%'}}
+        >
+          <div style={{width: '100%'}}>Name your function: </div>
```

Comment:
Nit: Are these placeholder strings for now? Just wondering about using I18n strings.

Reply:
Yes this will be improved later :)